common/dynamicconfig/constants.go

```diff
@@ -1260,6 +1260,12 @@ these log lines can be noisy, we want to be able to turn on and sample selective
 		1000,
 		`MatchingMaxTaskQueuesInDeployment represents the maximum number of task-queues that can be registed in a single deployment`,
 	)
+	DeploymentScavengerDuration = NewNamespaceDurationSetting(
+		"deployment.scavenger.duration",
```

Comment:
something like "matching.DeploymentScavengerDuration" to make it consistent with other configs?

Reply:
also probably should be called something like DeploymentScavengerRetryInterval to match its purpose.